{
  final Future<V> f=replaceAsyncInternal(key,null,value,expiryPolicy,false,true,true);
  try {
    return toObject(f.get());
  }
 catch (  Throwable e) {
    throw ExceptionUtil.rethrowAllowedTypeFirst(e,CacheException.class);
  }
}
